src/lib.rs

```diff
@@ -1242,7 +1242,7 @@ mod test {
                     },
                     #[cfg(feature = "esplora")]
                     esplora_opts: EsploraOpts {
-                        server: "https://blockstream.info/api/".to_string(),
+                        server: "https://blockstream.info/testnet/api/".to_string(),
```

Comment:
This seems redundant. Maybe we can remove this too? This is not activated in `electrum` tests and we don't have an `esplora` feature definition anymore.